modules/oc-mirror-installing-v2.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-oc-mirror-plugin-v2_{context}"]
+= Installing the oc-mirror plugin V2
```

Comment:
Again, this file is very similar to the steps for the original oc-mirror version. I'd recommend that you share that module and using conditionals to change any differences. Especially since it seems like it's the same binary, just that you pass in a --v2 flag to use the second version of it.